client/src/components/History/CurrentHistory/HistoryOperations/OperationErrorDialog.vue

```diff
@@ -1,3 +1,53 @@
+<script setup lang="ts">
+import { BModal } from "bootstrap-vue";
+import { computed, ref } from "vue";
+
+interface Props {
+    operationError: any;
```

Comment:
The type you are looking for should be around `HistoryContentBulkOperationResult`.

Reply:
It is something like:
```suggestion
    operationError: OperationError;
```
Where OperationError is:
```ts
interface OperationError {
    errorMessage: string;
    result: HistoryContentBulkOperationResult;
}
```
and HistoryContentBulkOperationResult comes from the API schema.